src/pretix/static/pretixbase/js/asynctask.js

```diff
@@ -275,21 +247,65 @@ $(function () {
             }, 10);
         }
     }, false);
+
+    $("#ajaxerr").on("click", ".ajaxerr-close", ajaxErrDialog.hide);
+    $("#loadingmodal").on("cancel", function() {
+        return false;
+    });
```

Comment:
This only works in Firefox, but causes a problem anyways: when I hit the Escape key, while the Ajax-request is still running, Firefox aborts any Ajax-requests running, but now does not close the modal either. Luckily the Ajax-request is still executing the success-callback when canceled through Firefox, but with different than expected results. So on success, we should always remove the loading-dialog.

In Chrome, blocking the Escape-key only works once. Hitting Escape a second time closes the dialog. It does not abort the Ajax-request though. So we need to check the `returnValue` on `dialog.close` whether the dialog was canceled and re-open the dialog if the Ajax-request is still loading.